### ISSUE TITLE
Report who and why turned off notifications

### ISSUE THREAD

**Issue by litleleprikon** [moira-alert/moira#469]
In Moira when notifications turned off is very hard to determine who and why turned off notifications. I propose to add a text field that should be filled with purpose when someone will switch off notifications. When notifications turned off Moira should display user who disabled notifications and purpose.

**Comment by idoqo** [comment]
Hi @litleleprikon, I'd love to work on this if it's still available. I'm not so clear if the notifications here refer to Subscriptions though.

**Comment by hv7214** [comment]
@idoqo are you working on this ?

**Comment by idoqo** [comment]
Oh No, not yet. Wasn't sure if the notification meant turning off a trigger or deleting a subscription.

**Comment by vrongmeal** [comment]
Hey! I'm new here. Can I take this up?

**Comment by litleleprikon** [comment]
Hi @idoqo notifications turned off means that this switcher on `/notifications` page is switched off

<img width="225" alt="Screenshot 2020-02-21 at 11 14 54" src="https://user-images.githubusercontent.com/5058463/75016271-a7cb3d00-549b-11ea-8834-af66a257bdd8.png">

**Comment by hv7214** [comment]
@litleleprikon can i pick this up ?

**Comment by idoqo** [comment]
> Hi @idoqo notifications turned off means that this switcher on `/notifications` page is switched off
> 
> <img alt="Screenshot 2020-02-21 at 11 14 54" width="225" src="https://user-images.githubusercontent.com/5058463/75016271-a7cb3d00-549b-11ea-8834-af66a257bdd8.png">

Okay got it, will be on it then.

**Comment by shashikant231** [comment]
Hi @litleleprikon  can you describe  some steps to reproduce this behaviour ?

**Comment by imshubh4141** [comment]
@idoqo Is the issue still open?

**Comment by idoqo** [comment]
Yeah it is, there's an incomplete implementation here https://github.com/moira-alert/moira/pull/520 so feel free to look at it for if you need some pointers.
